app/views/shared/_navigation.html.erb

```diff
@@ -1,7 +1,7 @@
 <nav class="navbar navbar-default">
   <div class="container">
     <ul class="nav navbar-nav">
-      <li><%= link_to "Wizlist", root_path, class: "navbar-brand" -%></li>
+      <li><%= link_to "Wizlist", root_path, class: "navbar-brand", id:"header-logo" -%></li>
```

Comment:
- you should use class here, `ID` is reserved for `js`
- missing space after id: